create-bastion.yml

```diff
@@ -0,0 +1,60 @@
+---
+- name: Provision and configure a bastion admin node
+  hosts: localhost
+  connection: local
+  gather_facts: False
+
+  vars_files:
+    - "{{ VARS_FILE }}"
+
+  tasks:
+    - name: Launch Bastion Node
+      local_action:
+        module: ec2
+        image: "{{ BASTION_IMAGE }}"
+        instance_type: "{{ BASTION_INSTANCE_TYPE }}"
+        keypair: "{{ KEYPAIR }}"
+        region: "{{ EC2_REGION }}"
+        group_id: "{{ BASTION_SG }}"
+        vpc_subnet_id: "{{ BASTION_SN }}"
+        assign_public_ip: yes
+        instance_tags:
+          Name: "Lightbend Admin Bastion"
+          Role: Bastion Node
+        count: 1
+        volumes:
+          - device_name: /dev/sda1
+            device_type: "{{ BASTION_VOL_TYPE }}"
+            volume_size: "{{ BASTION_VOL_SIZE }}"
+            delete_on_termination: true
+        wait: yes
+      register: ec2
+
+    - name: Add to bastion_public
+      add_host:
+        groupname: "bastion_public"
+        hostname: "{{ ec2.instances[0].public_ip }}"
+
+    - name: Wait for SSH to come up
+      wait_for:
+        host: "{{ groups.bastion_public[0] }}"
+        port: 22
+        delay: 60
+        timeout: 320
+        state: started
+
+- name: Setup Bastion
+  hosts: bastion_public
+  user: "ubuntu"
+  gather_facts: False
+  sudo: True
+
+  vars:
+   REMOTE_USER: "ubuntu"
+
+  tasks:
+    - include: python/tasks/main.yml
+    - include: system/tasks/update-ubuntu.yml
+    - include: ntp/tasks/main.yml
+    - include: ansible/tasks/setup-ansible.yml
+    - include: system/tasks/upgrade-ubuntu.yml
```

Comment:
@edc - do we want to install CLI on the bastion, or do we expect our user to control the cluster from one of the ConductR node?

I think if we were to control the cluster from Bastion, we can make it so that the Bastion only have outgoing access to the internet to download the bundles to be loaded into ConductR. This way ConductR cores don't require internet access (not sure if ConductR free will change this - afaik it shouldn't).

I think we do have the CLI on the Ansible bastion host at present, and I'm using it to load bundles to the clusters I built.

If we were to decide to load the bundles from the bastion host, I think we can refactor out these lines into separate `.yml` file and then call it from this playbook and the `install-core.yml` playbook:

https://github.com/typesafehub/conductr-ansible/blob/master/conductr/tasks/install-core.yml#L10-L33

It's another separate piece of work, so if you do agree it's a good thing to do, I'm quite happy for it to be a separate PR.